README.md

```diff
@@ -126,9 +127,33 @@ env:
   value: 500m
 '
 
+# Quota override for my organization
+MY_CUSTOM_ORG_QUOTA='
+- id: <YOUR ORG HERE>
+  max_allowed_instances: 1
```

Comment:
I think with 1, someone can only create `instanceType.x1` units i.e they cannot create `.x2` etc
I'd suggest to increase this number to say 10 because I know people are going to be copy/pasting this.